scripts/data_prep/convert_delta_to_json.py

```diff
@@ -667,18 +663,10 @@ def fetch_DT(args: Namespace) -> None:
         'The name of the combined final jsonl that combines all partitioned jsonl',
     )
     args = parser.parse_args()
-    mosaicml_logger = maybe_create_mosaicml_logger()
-
-    try:
-        w = WorkspaceClient()
-        args.DATABRICKS_HOST = w.config.host
-        args.DATABRICKS_TOKEN = w.config.token
-
-        tik = time.time()
-        fetch_DT(args)
-        log.info(f'Elapsed time {time.time() - tik}')
+    w = WorkspaceClient()
```

Comment:
does this import need to be gated?

Reply:
I don't think so, we aren't currently in gating it in the script.